test/ash_rbac_test.exs

```diff
@@ -1,6 +1,8 @@
 defmodule AshRbacTest do
   use ExUnit.Case, async: true
 
+  doctest AshRbac.Fields.OptionTransformer
```

Comment:
Do we need this to execute the doctests in the `OptionTransformer`?

Reply:
I think so, at least I saw this when I googled doctests, maybe there is a better way that just executes all doctests? 8

Reply:
Yep, not sure either. The only thing that came to my mind: "perhaps it would be nice to have a separate file for doctests instead of putting it into this file?"

But not sure about that, too.
It could be confusing when the rest of this file has nothing to do with the doctest itself.